### ISSUE TITLE
#Requires Windows PowerShell 1.7.1.1 exception

### ISSUE THREAD

**Issue by lancegosby** [NikCharlebois/SharePointDSC.Reverse#58]
**Details of the scenario you try and problem that is occurring:**
Just trying to run the SharePointDSC.Reverse.ps1 script as per your README.
I upgraded WMF to get PowerShell 5.
I ran `Install-Module SharePointDSC`
I didn't find the .psm1 file in your repo so I just installed the ReverseDSC module via `Install-Module ReverseDSC`.
Then I tried to run the .ps1 script from your repo. I get an error saying:
```
The script 'SharePointDSC.Reverse.ps1' cannot be run because it contained a "#requires" statement for Windows PowerShell 1.7.1.1. The version of Windows PowerShell that is required by the script does not match the currently running version of Windows PowerShell 5.0.10586.117.
```

I had to change the following in SharePointDSC.Reverse.ps1:
```
#Requires -Module ReverseDSC -Version 1.7.1.1
#Requires -Module SharePointDSC -Version 1.7.0.0
```
to this:
```
#Requires -Modules @{ModuleName="ReverseDSC";ModuleVersion="1.7.1.1"},@{ModuleName="SharePointDSC";ModuleVersion="1.7.0.0"}
```

Then the script runs as expected.

**The DSC configuration that is using the resource:**
Sorry. Not sure what this is asking for.

**Version of the Operating System and PowerShell the DSC Target Node is running:**
Server 2012 R2
PowerShell 5

**Version of the DSC module you're using:**
SharePointDSC: 1.7.0.0
ReverseDSC: 1.7.1.1

**Comment by NikCharlebois** [comment]
Thanks Lance. The latest version had that fixed. Just waiting to synchronize it with the main SharePointDSC branch release.

**Comment by NikCharlebois** [comment]
Fixed in the new 1.8.0.0 Release
